application/static/validation.css

```diff
@@ -6,10 +6,50 @@
   width: 75%;
 }
 
+
```

Comment:
Is copying really the solution or (a) can we separate it out into a different file. Or (b) unify all styles in a single file. That's for later obviously.